packages/map/package.json

```diff
@@ -6,9 +6,7 @@
     "dev": "next dev",
     "build": "next build && next export -o build",
     "start": "next start",
-    "lint": "eslint .",
-    "lint:fix": "eslint --fix .",
```

Comment:
I would appreciate some context why `lint:fix` is being removed, can you please explain?

Reply:
Since version 11.0.0, Next.js provides an integrated ESLint [out of the box](https://nextjs.org/docs/basic-features/eslint). I tried to use `next lint --fix .` which didn't work. But now I figured out that we can still use `eslint --fix .` because it's included inside next's eslint.
In next commit I will return `lint:fix` and prettier configuration.